### DISCUSSION TITLE
HmIP-BDT Dimmer - creates only switch

### DISCUSSION THREAD

**Discussion by kstuken** [SukramJ/aiohomematic#1944] [Q&A] [answered]
Hi, I have just started my journey transitioning from openHAB to HA. (And hence there is a chance I am missing something quite obvious, as I am still relatively inexperienced with HA; apologies in advance if that is the case).
Thank you for this integration, it is essential for my transition.

Now, I have a  HmIP-BDT Dimmer. The integration properly found it, but only mapped it to a switch, ie on/off. So I can turn this light on and off, but I cannot change the brightness. I cannot seem to find any way to change the brightness. I can see entities for channels 3 (read brightness), 5, and 6 (both unused), but none for channel 4 (which controls the brightness). I assume the on/off switch is mapped to channel 4. But I would rather want a dimmer slider from 0 to 100.

So I have two quick questions:
* is this a bug in the integration, or expected behavior?
* is there a straightforward way to fix this in my system?

Thank you!
Kai.

**Comment by SukramJ** [comment]
Please post a screenshot.

**Reply by kstuken** [reply, marked as answer]
First of all, thanks for the immediate response.
And sorry, it seems this is more a combination of my inexperience with HA plus the somewhat odd behavior of HA with this dimmer. I noticed that I can actually slide the large switch that pops up when I click on the entity name. So basically the slider/dimmer functionality is there. What does irritate me, though, is that if the dimmer is set to eg 20%, the on/off switch shown in eg the device or the dashboard still shows "off" rather than "on", which is non-intuitive. But I guess this is a HA issue rather than an issue with this integration.

Sorry to waste your time!

**Reply by SukramJ** [reply]
Thats what i expected.
